pkg/webui/console/components/pubsub-form/validation-schema.js

```diff
@@ -34,7 +34,7 @@ export default Yup.object().shape({
     is: 'nats',
     then: Yup.object().shape({
       username: Yup.string()
-        .matches(idRegexp, sharedMessages.validateAlphanum)
+        .matches(idRegexp, sharedMessages.validateIdFormat)
```

Comment:
It doesn't really fit here, as it would say `IDs can only contain…`. Let's change the message to be more generic: `This value may only contain…`